tracing-attributes/src/lib.rs

```diff
@@ -853,20 +864,67 @@ impl Parse for Level {
     }
 }
 
-fn param_names(pat: Pat) -> Box<dyn Iterator<Item = Ident>> {
+/// A enum indicates the field should be recorded as `Value` or `Debug`.
```

Comment:
nit: mayberephrase this to just
```suggestion
/// Indicates whether a field should be recorded as `Value` or `Debug`.
```